packages/protocol/contracts/identity/test/AttestationsTest.sol

```diff
@@ -21,4 +21,181 @@ contract AttestationsTest is Attestations(true) {
   function validatorSignerAddressFromCurrentSet(uint256 index) public view returns (address) {
     return __testValidators[index];
   }
+
+  // reimplement deprecated nonview functions here for testing purposes:
```

Comment:
This is so that the protocol attestations tests can check that revoking/withdrawing works still, is that correct?

If that is, maybe you can add a bit more detail to this comment (took me a sec to understand why we would need to keep this) that it's specifically still necessary for revoke/withdraw?